Breaks compat with pre ~5
https://gitlab.com/ddcci-driver-linux/ddcci-driver-linux/-/commit/7f2c8e73e9e9f11466473ef971b4c583b4a3ece8

```
error: builder for '/nix/store/jl58jx619hgf7qbi2m15v0k0818w0hii-ddcci-driver-4.19.205-0.4.1.drv' failed with exit code 2;
       last 10 log lines:
       >    55 | static_assert(sizeof_field(struct ddcci_device, module) > 8);
       >       |               ^~~~~~~~~~~~
       > make[5]: *** [/nix/store/phz1jcdpywc01k4hs7jm8156vjvf5bzi-linux-4.19.205-dev/lib/modules/4.19.205-hardened1/source/scripts/Makefile.build
:310: /build/source/ddcci/ddcci.o] Error 1
       > make[4]: *** [/nix/store/phz1jcdpywc01k4hs7jm8156vjvf5bzi-linux-4.19.205-dev/lib/modules/4.19.205-hardened1/source/Makefile:1549: _module
_/build/source/ddcci] Error 2
       > make[3]: *** [Makefile:146: sub-make] Error 2
       > make[2]: *** [Makefile:24: __sub-make] Error 2
       > make[2]: Leaving directory '/nix/store/phz1jcdpywc01k4hs7jm8156vjvf5bzi-linux-4.19.205-dev/lib/modules/4.19.205-hardened1/build'
       > make[1]: *** [Makefile:38: ddcci.ko] Error 2
       > make[1]: Leaving directory '/build/source/ddcci'
       > make: *** [Makefile:28: ddcci] Error 2
       For full logs, run 'nix log /nix/store/jl58jx619hgf7qbi2m15v0k0818w0hii-ddcci-driver-4.19.205-0.4.1.
```

--- a/pkgs/os-specific/linux/ddcci/default.nix
+++ b/pkgs/os-specific/linux/ddcci/default.nix
@@ -2,14 +2,14 @@
 
 stdenv.mkDerivation rec {
   pname = "ddcci-driver";
-  version = "0.3.3";
+  version = "0.4.1";
   name = "${pname}-${kernel.version}-${version}";
 
   src = fetchFromGitLab {
     owner = "${pname}-linux";
     repo = "${pname}-linux";
     rev = "v${version}";
-    sha256 = "0vkkja3ykjil783zjpwp0vz7jy2fp9ccazzi3afd4fjk8gldin7f";
+    sha256 = "1qhsm0ccwfmwn0r6sbc6ms4lf4a3iqfcgqmbs6afr6hhxkqll3fg";
   };
 
   hardeningDisable = [ "pic" ];
@@ -35,8 +35,9 @@ stdenv.mkDerivation rec {
   meta = with lib; {
     description = "Kernel module driver for DDC/CI monitors";
     homepage = "https://gitlab.com/ddcci-driver-linux/ddcci-driver-linux";
-    license = licenses.gpl2;
+    license = licenses.gpl2Plus;
     maintainers = with maintainers; [ bricewge ];
     platforms = platforms.linux;
+    broken = kernel.kernelOlder "5.1";
   };
 }

